index.html

```diff
@@ -1,298 +1,341 @@
+                            <a href="#" class="M"><img src="images/M.svg"
+                                    alt=""></a>
+                        </li>
+                        <li>
+                            <a href="#" class="C"><img src="images/C.svg"
+                                    alt=""></a>
+                        </li>
+                        <li>
+                            <a href="#" class="2f"><img src="images/2f.svg"
+                                    alt=""></a>
                         </li>
                     </ul>
                 </div>
-                <div class="see-project">
-                    <div><a class="button-project" href="#">See Project</a></div>
-                </div>
 
             </div>
-        </main>
+            <div class="view-resume">
+                <div><a class="button-project" href="#">Get my resume</a></div>
+            </div>
+        </div>
 
-       <section>
-        <div class="container-about">
-            <div class="About-Myself">
-                <h2>About
-                    Myself</h2>
+        <div class="lang-select">
+            <div class="language-label">Languages</div>
+            <div class="lang-arrow-button"><a href="#"><img alt=" "
+                        src="images/icons8-expand-arrow-30.png"></a></div>
+        </div>
+
+        <div class="lang-box-container">
+
+            <div class="lang-box">
+                <div class="lang-logo-js"></div>
+                <div class="lang-label">javascript</div>
             </div>
-            <div class="intro-desc-sumary">
-                <p>Hello I’m a software developer! 
-                I can help you build a product
-                 , feature or website Look through some 
-                 of my work and experience!
-                  If you like what you see and have a project 
-                  you need coded, don’t hestiate to contact me.</p>
+
+            <div class="lang-box">
+                <div class="lang-logo-html"></div>
+                <div class="lang-label">HTML</div>
             </div>
-        
 
-        <div class="social-area">
-            <h4 class="social-network">LET’S CONNECT</h4>
-            <div class="social-icon">
+            <div class="lang-box">
+                <div class="lang-logo-css"></div>
+                <div class="lang-label">CSS</div>
+            </div>
+
+        </div>
+        <div class="framework-select">
+            <div class="framework-label">Framework</div>
+            <div class="frameW-arrow-button"><a href="#"><img alt=" "
+                        src="images/icons8-right-arrow-32.png"></a></div>
+        </div>
+
+        <div class="skills-select">
+            <div class="skills-label">Skills</div>
+            <div class="skills-arrow-button"><a href="#"><img alt=" "
+                        src="images/icons8-right-arrow-32.png"></a></div>
+        </div>
+
+    </section>
+    <section>
+        <div class="form-container">
```

Comment:
Optional:

- [ ] Your contact form is not matched with the Figma design. Your page has the whitespace on the right side of the contact form and at the end of the contact form, there should be an underline as the Figma design has.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![c1](https://user-images.githubusercontent.com/70483835/165364278-b5a491b0-5971-443b-93a8-05b56778a397.png) | ![c2](https://user-images.githubusercontent.com/70483835/165364295-c0dc5709-5c1c-42a7-ac54-4918d83f3883.png)